includes/class-wc-siftscience-order-status.php

```diff
@@ -0,0 +1,45 @@
+<?php
+/**
+ * This class handles the display of Sift feedback icons in order list and order view.
+ *
+ * @author Nabeel Sulieman, Rami Jamleh
+ * @package sift-for-woocommerce
+ * @license GPL2
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+if ( ! class_exists( 'WC_SiftScience_Order_Status' ) ) :
+
+	/**
+	 * Class WC_SiftScience_Orders
+	 */
+	class WC_SiftScience_Order_Status {
+
+		/**
+		 * Cache of order status options to avoid fetching it too many times
+		 *
+		 * @var array List of status options
+		 */
+		private $statuses = null;
+
+		/**
+		 * Gets a list of available statuses
+		 *
+		 * @return array The full list of statuses and order can have.
+		 */
+		public function get_status_options() {
+			if ( null === $this->statuses ) {
+				$this->statuses = array( 'none' => 'Do Nothing' );
+				foreach ( wc_get_order_statuses() as $key => $val ) {
+					$this->statuses[ $key ] = $val;
+				}
+			}
+
+			return $this->statuses;
```

Comment:
it's a long day 
hear me up 
1. states should have IDs for further scaling 
2. the logic inside the function better left in the __construct 
3. a separate getter for statuses should return the filled array not looping every time we get states

Reply:
1: It's not in the constructor because we might not always use the list of statuses (when I add more functions this will be more obvious).

2: The loop is inside an if-statement, so it only gets run once.

Reply:
Then simply we add the add/remove functions

Reply:
I don't understand.